core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -1360,7 +1360,6 @@ impl SubstateDatabase for RocksDBStore {
             self.open_db_context()
                 .cf(SubstatesCf)
                 .iterate_group_from(&(partition_key.clone(), from_sort_key), Direction::Forward)
-                .take_while(move |((next_key, _), _)| next_key == &partition_key)
```

Comment:
_(note from author)_

This is just a drive-by mentioned at https://rdxworks.slack.com/archives/D04CPLNN81Y/p1705591790950219